package.json

```diff
@@ -1,7 +1,7 @@
 {
   "name": "kuzzle",
   "author": "The Kuzzle Team <support@kuzzle.io>",
-  "version": "1.7.4",
+  "version": "1.7.5",
```

Comment:
you have to update the version stored in `package-lock.json` too, otherwise it's ignored

instead of updating the version number manually, you should have used `npm version [major|minor|patch]`

Reply:
You can also generate a new `package-lock.json` file by removing the old one and run `npm install`